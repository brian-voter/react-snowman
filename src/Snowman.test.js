import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Snowman from "./Snowman";

describe("only allows maxWrong incorrect guesses", function () {
  it("letter buttons are hidden after making maxWrong guesses", function () {
    const word = "apple";
    const maxWrong = 6; //TODO: replace this

    const { container } = render(
      <Snowman words={[word]} maxWrong={maxWrong} />
    );

    const wrongLetters = "abcdefghijklmnopqrstuvwxyz"
      .split("")
      .filter((l) => !word.includes(l));

    const wrongButtons = wrongLetters.map((l) =>
      container.querySelector(`button[value="${l}"]`)
    );

    for (let i = 0; i < maxWrong; i++) {
      fireEvent.click(wrongButtons[i]);
    }

    const snowmanImage = container.querySelector(`img`);
    const imgAlt = snowmanImage.getAttribute("alt");

    expect(imgAlt).toEqual(String(maxWrong));

    const letterButtonContainer = container.querySelector(
      "#letterButtonContainer"
    );

    expect(letterButtonContainer).toHaveClass("hidden");
    expect(container).toMatchSnapshot();
  });
});
