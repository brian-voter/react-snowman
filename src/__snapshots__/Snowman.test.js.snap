// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`only allows maxWrong incorrect guesses image alt should  1`] = `
<div>
  <div
    class="Snowman"
  >
    <img
      alt="0"
      src="0.png"
    />
    <p>
      Number wrong: 
      0
    </p>
    <p
      class="Snowman-word"
    >
      _
      _
      _
      _
      _
    </p>
    <p>
      <button
        value="a"
      >
        a
      </button>
      <button
        value="b"
      >
        b
      </button>
      <button
        value="c"
      >
        c
      </button>
      <button
        value="d"
      >
        d
      </button>
      <button
        value="e"
      >
        e
      </button>
      <button
        value="f"
      >
        f
      </button>
      <button
        value="g"
      >
        g
      </button>
      <button
        value="h"
      >
        h
      </button>
      <button
        value="i"
      >
        i
      </button>
      <button
        value="j"
      >
        j
      </button>
      <button
        value="k"
      >
        k
      </button>
      <button
        value="l"
      >
        l
      </button>
      <button
        value="m"
      >
        m
      </button>
      <button
        value="n"
      >
        n
      </button>
      <button
        value="o"
      >
        o
      </button>
      <button
        value="p"
      >
        p
      </button>
      <button
        value="q"
      >
        q
      </button>
      <button
        value="r"
      >
        r
      </button>
      <button
        value="s"
      >
        s
      </button>
      <button
        value="t"
      >
        t
      </button>
      <button
        value="u"
      >
        u
      </button>
      <button
        value="v"
      >
        v
      </button>
      <button
        value="w"
      >
        w
      </button>
      <button
        value="x"
      >
        x
      </button>
      <button
        value="y"
      >
        y
      </button>
      <button
        value="z"
      >
        z
      </button>
    </p>
  </div>
</div>
`;

exports[`only allows maxWrong incorrect guesses image alt should be equal to maxWrong after maxWrong +  1`] = `
<div>
  <div
    class="Snowman"
  >
    <img
      alt="0"
      src="0.png"
    />
    <p>
      Number wrong: 
      0
    </p>
    <p
      class="Snowman-word"
    >
      _
      _
      _
      _
      _
    </p>
    <p>
      <button
        value="a"
      >
        a
      </button>
      <button
        value="b"
      >
        b
      </button>
      <button
        value="c"
      >
        c
      </button>
      <button
        value="d"
      >
        d
      </button>
      <button
        value="e"
      >
        e
      </button>
      <button
        value="f"
      >
        f
      </button>
      <button
        value="g"
      >
        g
      </button>
      <button
        value="h"
      >
        h
      </button>
      <button
        value="i"
      >
        i
      </button>
      <button
        value="j"
      >
        j
      </button>
      <button
        value="k"
      >
        k
      </button>
      <button
        value="l"
      >
        l
      </button>
      <button
        value="m"
      >
        m
      </button>
      <button
        value="n"
      >
        n
      </button>
      <button
        value="o"
      >
        o
      </button>
      <button
        value="p"
      >
        p
      </button>
      <button
        value="q"
      >
        q
      </button>
      <button
        value="r"
      >
        r
      </button>
      <button
        value="s"
      >
        s
      </button>
      <button
        value="t"
      >
        t
      </button>
      <button
        value="u"
      >
        u
      </button>
      <button
        value="v"
      >
        v
      </button>
      <button
        value="w"
      >
        w
      </button>
      <button
        value="x"
      >
        x
      </button>
      <button
        value="y"
      >
        y
      </button>
      <button
        value="z"
      >
        z
      </button>
    </p>
  </div>
</div>
`;

exports[`only allows maxWrong incorrect guesses image alt should be equal to maxWrong after maxWrong + 1 guesses 1`] = `
<div>
  <div
    class="Snowman"
  >
    <img
      alt="0"
      src="0.png"
    />
    <p>
      Number wrong: 
      0
    </p>
    <p
      class="Snowman-word"
    >
      _
      _
      _
      _
      _
    </p>
    <p>
      <button
        value="a"
      >
        a
      </button>
      <button
        value="b"
      >
        b
      </button>
      <button
        value="c"
      >
        c
      </button>
      <button
        value="d"
      >
        d
      </button>
      <button
        value="e"
      >
        e
      </button>
      <button
        value="f"
      >
        f
      </button>
      <button
        value="g"
      >
        g
      </button>
      <button
        value="h"
      >
        h
      </button>
      <button
        value="i"
      >
        i
      </button>
      <button
        value="j"
      >
        j
      </button>
      <button
        value="k"
      >
        k
      </button>
      <button
        value="l"
      >
        l
      </button>
      <button
        value="m"
      >
        m
      </button>
      <button
        value="n"
      >
        n
      </button>
      <button
        value="o"
      >
        o
      </button>
      <button
        value="p"
      >
        p
      </button>
      <button
        value="q"
      >
        q
      </button>
      <button
        value="r"
      >
        r
      </button>
      <button
        value="s"
      >
        s
      </button>
      <button
        value="t"
      >
        t
      </button>
      <button
        value="u"
      >
        u
      </button>
      <button
        value="v"
      >
        v
      </button>
      <button
        value="w"
      >
        w
      </button>
      <button
        value="x"
      >
        x
      </button>
      <button
        value="y"
      >
        y
      </button>
      <button
        value="z"
      >
        z
      </button>
    </p>
  </div>
</div>
`;

exports[`only allows maxWrong incorrect guesses letter buttons are hidden after guessing  maxWrong after maxWrong + 1 guesses 1`] = `
<div>
  <div
    class="Snowman"
  >
    <img
      alt="6"
      src="6.png"
    />
    <p>
      Number wrong: 
      6
    </p>
    <p
      class="Snowman-word"
    >
      _
      _
      _
      _
      _
    </p>
    <div>
      <p>
        You lose! 
      </p>
      <p>
        apple
      </p>
    </div>
    <p
      class="hidden"
      id="letterButtonContainer"
    >
       
      <button
        value="a"
      >
        a
      </button>
      <button
        disabled=""
        value="b"
      >
        b
      </button>
      <button
        disabled=""
        value="c"
      >
        c
      </button>
      <button
        disabled=""
        value="d"
      >
        d
      </button>
      <button
        value="e"
      >
        e
      </button>
      <button
        disabled=""
        value="f"
      >
        f
      </button>
      <button
        disabled=""
        value="g"
      >
        g
      </button>
      <button
        disabled=""
        value="h"
      >
        h
      </button>
      <button
        value="i"
      >
        i
      </button>
      <button
        value="j"
      >
        j
      </button>
      <button
        value="k"
      >
        k
      </button>
      <button
        value="l"
      >
        l
      </button>
      <button
        value="m"
      >
        m
      </button>
      <button
        value="n"
      >
        n
      </button>
      <button
        value="o"
      >
        o
      </button>
      <button
        value="p"
      >
        p
      </button>
      <button
        value="q"
      >
        q
      </button>
      <button
        value="r"
      >
        r
      </button>
      <button
        value="s"
      >
        s
      </button>
      <button
        value="t"
      >
        t
      </button>
      <button
        value="u"
      >
        u
      </button>
      <button
        value="v"
      >
        v
      </button>
      <button
        value="w"
      >
        w
      </button>
      <button
        value="x"
      >
        x
      </button>
      <button
        value="y"
      >
        y
      </button>
      <button
        value="z"
      >
        z
      </button>
    </p>
  </div>
</div>
`;

exports[`only allows maxWrong incorrect guesses letter buttons are hidden after guessing maxWrong after maxWrong + 1 guesses 1`] = `
<div>
  <div
    class="Snowman"
  >
    <img
      alt="6"
      src="6.png"
    />
    <p>
      Number wrong: 
      6
    </p>
    <p
      class="Snowman-word"
    >
      _
      _
      _
      _
      _
    </p>
    <div>
      <p>
        You lose! 
      </p>
      <p>
        apple
      </p>
    </div>
    <p
      class="hidden"
      id="letterButtonContainer"
    >
       
      <button
        value="a"
      >
        a
      </button>
      <button
        disabled=""
        value="b"
      >
        b
      </button>
      <button
        disabled=""
        value="c"
      >
        c
      </button>
      <button
        disabled=""
        value="d"
      >
        d
      </button>
      <button
        value="e"
      >
        e
      </button>
      <button
        disabled=""
        value="f"
      >
        f
      </button>
      <button
        disabled=""
        value="g"
      >
        g
      </button>
      <button
        disabled=""
        value="h"
      >
        h
      </button>
      <button
        value="i"
      >
        i
      </button>
      <button
        value="j"
      >
        j
      </button>
      <button
        value="k"
      >
        k
      </button>
      <button
        value="l"
      >
        l
      </button>
      <button
        value="m"
      >
        m
      </button>
      <button
        value="n"
      >
        n
      </button>
      <button
        value="o"
      >
        o
      </button>
      <button
        value="p"
      >
        p
      </button>
      <button
        value="q"
      >
        q
      </button>
      <button
        value="r"
      >
        r
      </button>
      <button
        value="s"
      >
        s
      </button>
      <button
        value="t"
      >
        t
      </button>
      <button
        value="u"
      >
        u
      </button>
      <button
        value="v"
      >
        v
      </button>
      <button
        value="w"
      >
        w
      </button>
      <button
        value="x"
      >
        x
      </button>
      <button
        value="y"
      >
        y
      </button>
      <button
        value="z"
      >
        z
      </button>
    </p>
  </div>
</div>
`;

exports[`only allows maxWrong incorrect guesses letter buttons are hidden after guessing maxWrong times 1`] = `
<div>
  <div
    class="Snowman"
  >
    <img
      alt="6"
      src="6.png"
    />
    <p>
      Number wrong: 
      6
    </p>
    <p
      class="Snowman-word"
    >
      _
      _
      _
      _
      _
    </p>
    <div>
      <p>
        You lose! 
      </p>
      <p>
        apple
      </p>
    </div>
    <p
      class="hidden"
      id="letterButtonContainer"
    >
       
      <button
        value="a"
      >
        a
      </button>
      <button
        disabled=""
        value="b"
      >
        b
      </button>
      <button
        disabled=""
        value="c"
      >
        c
      </button>
      <button
        disabled=""
        value="d"
      >
        d
      </button>
      <button
        value="e"
      >
        e
      </button>
      <button
        disabled=""
        value="f"
      >
        f
      </button>
      <button
        disabled=""
        value="g"
      >
        g
      </button>
      <button
        disabled=""
        value="h"
      >
        h
      </button>
      <button
        value="i"
      >
        i
      </button>
      <button
        value="j"
      >
        j
      </button>
      <button
        value="k"
      >
        k
      </button>
      <button
        value="l"
      >
        l
      </button>
      <button
        value="m"
      >
        m
      </button>
      <button
        value="n"
      >
        n
      </button>
      <button
        value="o"
      >
        o
      </button>
      <button
        value="p"
      >
        p
      </button>
      <button
        value="q"
      >
        q
      </button>
      <button
        value="r"
      >
        r
      </button>
      <button
        value="s"
      >
        s
      </button>
      <button
        value="t"
      >
        t
      </button>
      <button
        value="u"
      >
        u
      </button>
      <button
        value="v"
      >
        v
      </button>
      <button
        value="w"
      >
        w
      </button>
      <button
        value="x"
      >
        x
      </button>
      <button
        value="y"
      >
        y
      </button>
      <button
        value="z"
      >
        z
      </button>
    </p>
  </div>
</div>
`;

exports[`only allows maxWrong incorrect guesses letter buttons are hidden after guessing maxWrong timesafter maxWrong + 1 guesses 1`] = `
<div>
  <div
    class="Snowman"
  >
    <img
      alt="6"
      src="6.png"
    />
    <p>
      Number wrong: 
      6
    </p>
    <p
      class="Snowman-word"
    >
      _
      _
      _
      _
      _
    </p>
    <div>
      <p>
        You lose! 
      </p>
      <p>
        apple
      </p>
    </div>
    <p
      class="hidden"
      id="letterButtonContainer"
    >
       
      <button
        value="a"
      >
        a
      </button>
      <button
        disabled=""
        value="b"
      >
        b
      </button>
      <button
        disabled=""
        value="c"
      >
        c
      </button>
      <button
        disabled=""
        value="d"
      >
        d
      </button>
      <button
        value="e"
      >
        e
      </button>
      <button
        disabled=""
        value="f"
      >
        f
      </button>
      <button
        disabled=""
        value="g"
      >
        g
      </button>
      <button
        disabled=""
        value="h"
      >
        h
      </button>
      <button
        value="i"
      >
        i
      </button>
      <button
        value="j"
      >
        j
      </button>
      <button
        value="k"
      >
        k
      </button>
      <button
        value="l"
      >
        l
      </button>
      <button
        value="m"
      >
        m
      </button>
      <button
        value="n"
      >
        n
      </button>
      <button
        value="o"
      >
        o
      </button>
      <button
        value="p"
      >
        p
      </button>
      <button
        value="q"
      >
        q
      </button>
      <button
        value="r"
      >
        r
      </button>
      <button
        value="s"
      >
        s
      </button>
      <button
        value="t"
      >
        t
      </button>
      <button
        value="u"
      >
        u
      </button>
      <button
        value="v"
      >
        v
      </button>
      <button
        value="w"
      >
        w
      </button>
      <button
        value="x"
      >
        x
      </button>
      <button
        value="y"
      >
        y
      </button>
      <button
        value="z"
      >
        z
      </button>
    </p>
  </div>
</div>
`;

exports[`only allows maxWrong incorrect guesses letter buttons are hidden after image alt should be equal to maxWrong after maxWrong + 1 guesses 1`] = `
<div>
  <div
    class="Snowman"
  >
    <img
      alt="6"
      src="6.png"
    />
    <p>
      Number wrong: 
      6
    </p>
    <p
      class="Snowman-word"
    >
      _
      _
      _
      _
      _
    </p>
    <div>
      <p>
        You lose! 
      </p>
      <p>
        apple
      </p>
    </div>
    <p
      class="hidden"
      id="letterButtonContainer"
    >
       
      <button
        value="a"
      >
        a
      </button>
      <button
        disabled=""
        value="b"
      >
        b
      </button>
      <button
        disabled=""
        value="c"
      >
        c
      </button>
      <button
        disabled=""
        value="d"
      >
        d
      </button>
      <button
        value="e"
      >
        e
      </button>
      <button
        disabled=""
        value="f"
      >
        f
      </button>
      <button
        disabled=""
        value="g"
      >
        g
      </button>
      <button
        disabled=""
        value="h"
      >
        h
      </button>
      <button
        value="i"
      >
        i
      </button>
      <button
        value="j"
      >
        j
      </button>
      <button
        value="k"
      >
        k
      </button>
      <button
        value="l"
      >
        l
      </button>
      <button
        value="m"
      >
        m
      </button>
      <button
        value="n"
      >
        n
      </button>
      <button
        value="o"
      >
        o
      </button>
      <button
        value="p"
      >
        p
      </button>
      <button
        value="q"
      >
        q
      </button>
      <button
        value="r"
      >
        r
      </button>
      <button
        value="s"
      >
        s
      </button>
      <button
        value="t"
      >
        t
      </button>
      <button
        value="u"
      >
        u
      </button>
      <button
        value="v"
      >
        v
      </button>
      <button
        value="w"
      >
        w
      </button>
      <button
        value="x"
      >
        x
      </button>
      <button
        value="y"
      >
        y
      </button>
      <button
        value="z"
      >
        z
      </button>
    </p>
  </div>
</div>
`;

exports[`only allows maxWrong incorrect guesses letter buttons are hidden after making maxWrong guesses 1`] = `
<div>
  <div
    class="Snowman"
  >
    <img
      alt="6"
      src="6.png"
    />
    <p>
      Number wrong: 
      6
    </p>
    <p
      class="Snowman-word"
    >
      _
      _
      _
      _
      _
    </p>
    <div>
      <p>
        You lose! 
      </p>
      <p>
        apple
      </p>
    </div>
    <p
      class="hidden"
      id="letterButtonContainer"
    >
       
      <button
        value="a"
      >
        a
      </button>
      <button
        disabled=""
        value="b"
      >
        b
      </button>
      <button
        disabled=""
        value="c"
      >
        c
      </button>
      <button
        disabled=""
        value="d"
      >
        d
      </button>
      <button
        value="e"
      >
        e
      </button>
      <button
        disabled=""
        value="f"
      >
        f
      </button>
      <button
        disabled=""
        value="g"
      >
        g
      </button>
      <button
        disabled=""
        value="h"
      >
        h
      </button>
      <button
        value="i"
      >
        i
      </button>
      <button
        value="j"
      >
        j
      </button>
      <button
        value="k"
      >
        k
      </button>
      <button
        value="l"
      >
        l
      </button>
      <button
        value="m"
      >
        m
      </button>
      <button
        value="n"
      >
        n
      </button>
      <button
        value="o"
      >
        o
      </button>
      <button
        value="p"
      >
        p
      </button>
      <button
        value="q"
      >
        q
      </button>
      <button
        value="r"
      >
        r
      </button>
      <button
        value="s"
      >
        s
      </button>
      <button
        value="t"
      >
        t
      </button>
      <button
        value="u"
      >
        u
      </button>
      <button
        value="v"
      >
        v
      </button>
      <button
        value="w"
      >
        w
      </button>
      <button
        value="x"
      >
        x
      </button>
      <button
        value="y"
      >
        y
      </button>
      <button
        value="z"
      >
        z
      </button>
    </p>
  </div>
</div>
`;

exports[`only allows maxWrong incorrect guesses letter buttons are hidden after making maxWrong times 1`] = `
<div>
  <div
    class="Snowman"
  >
    <img
      alt="6"
      src="6.png"
    />
    <p>
      Number wrong: 
      6
    </p>
    <p
      class="Snowman-word"
    >
      _
      _
      _
      _
      _
    </p>
    <div>
      <p>
        You lose! 
      </p>
      <p>
        apple
      </p>
    </div>
    <p
      class="hidden"
      id="letterButtonContainer"
    >
       
      <button
        value="a"
      >
        a
      </button>
      <button
        disabled=""
        value="b"
      >
        b
      </button>
      <button
        disabled=""
        value="c"
      >
        c
      </button>
      <button
        disabled=""
        value="d"
      >
        d
      </button>
      <button
        value="e"
      >
        e
      </button>
      <button
        disabled=""
        value="f"
      >
        f
      </button>
      <button
        disabled=""
        value="g"
      >
        g
      </button>
      <button
        disabled=""
        value="h"
      >
        h
      </button>
      <button
        value="i"
      >
        i
      </button>
      <button
        value="j"
      >
        j
      </button>
      <button
        value="k"
      >
        k
      </button>
      <button
        value="l"
      >
        l
      </button>
      <button
        value="m"
      >
        m
      </button>
      <button
        value="n"
      >
        n
      </button>
      <button
        value="o"
      >
        o
      </button>
      <button
        value="p"
      >
        p
      </button>
      <button
        value="q"
      >
        q
      </button>
      <button
        value="r"
      >
        r
      </button>
      <button
        value="s"
      >
        s
      </button>
      <button
        value="t"
      >
        t
      </button>
      <button
        value="u"
      >
        u
      </button>
      <button
        value="v"
      >
        v
      </button>
      <button
        value="w"
      >
        w
      </button>
      <button
        value="x"
      >
        x
      </button>
      <button
        value="y"
      >
        y
      </button>
      <button
        value="z"
      >
        z
      </button>
    </p>
  </div>
</div>
`;

exports[`only allows maxWrong incorrect guesses letter buttons image alt should be equal to maxWrong after maxWrong + 1 guesses 1`] = `
<div>
  <div
    class="Snowman"
  >
    <img
      alt="6"
      src="6.png"
    />
    <p>
      Number wrong: 
      6
    </p>
    <p
      class="Snowman-word"
    >
      _
      _
      _
      _
      _
    </p>
    <div>
      <p>
        You lose! 
      </p>
      <p>
        apple
      </p>
    </div>
    <p
      class="hidden"
      id="letterButtonContainer"
    >
       
      <button
        value="a"
      >
        a
      </button>
      <button
        disabled=""
        value="b"
      >
        b
      </button>
      <button
        disabled=""
        value="c"
      >
        c
      </button>
      <button
        disabled=""
        value="d"
      >
        d
      </button>
      <button
        value="e"
      >
        e
      </button>
      <button
        disabled=""
        value="f"
      >
        f
      </button>
      <button
        disabled=""
        value="g"
      >
        g
      </button>
      <button
        disabled=""
        value="h"
      >
        h
      </button>
      <button
        value="i"
      >
        i
      </button>
      <button
        value="j"
      >
        j
      </button>
      <button
        value="k"
      >
        k
      </button>
      <button
        value="l"
      >
        l
      </button>
      <button
        value="m"
      >
        m
      </button>
      <button
        value="n"
      >
        n
      </button>
      <button
        value="o"
      >
        o
      </button>
      <button
        value="p"
      >
        p
      </button>
      <button
        value="q"
      >
        q
      </button>
      <button
        value="r"
      >
        r
      </button>
      <button
        value="s"
      >
        s
      </button>
      <button
        value="t"
      >
        t
      </button>
      <button
        value="u"
      >
        u
      </button>
      <button
        value="v"
      >
        v
      </button>
      <button
        value="w"
      >
        w
      </button>
      <button
        value="x"
      >
        x
      </button>
      <button
        value="y"
      >
        y
      </button>
      <button
        value="z"
      >
        z
      </button>
    </p>
  </div>
</div>
`;

exports[`only allows maxWrong incorrect guesses matches initial player hand: 2C 3C 1`] = `
<div>
  <div
    class="Snowman"
  >
    <img
      alt="0"
      src="0.png"
    />
    <p>
      Number wrong: 
      0
    </p>
    <p
      class="Snowman-word"
    >
      _
      _
      _
      _
      _
    </p>
    <p>
      <button
        value="a"
      >
        a
      </button>
      <button
        value="b"
      >
        b
      </button>
      <button
        value="c"
      >
        c
      </button>
      <button
        value="d"
      >
        d
      </button>
      <button
        value="e"
      >
        e
      </button>
      <button
        value="f"
      >
        f
      </button>
      <button
        value="g"
      >
        g
      </button>
      <button
        value="h"
      >
        h
      </button>
      <button
        value="i"
      >
        i
      </button>
      <button
        value="j"
      >
        j
      </button>
      <button
        value="k"
      >
        k
      </button>
      <button
        value="l"
      >
        l
      </button>
      <button
        value="m"
      >
        m
      </button>
      <button
        value="n"
      >
        n
      </button>
      <button
        value="o"
      >
        o
      </button>
      <button
        value="p"
      >
        p
      </button>
      <button
        value="q"
      >
        q
      </button>
      <button
        value="r"
      >
        r
      </button>
      <button
        value="s"
      >
        s
      </button>
      <button
        value="t"
      >
        t
      </button>
      <button
        value="u"
      >
        u
      </button>
      <button
        value="v"
      >
        v
      </button>
      <button
        value="w"
      >
        w
      </button>
      <button
        value="x"
      >
        x
      </button>
      <button
        value="y"
      >
        y
      </button>
      <button
        value="z"
      >
        z
      </button>
    </p>
  </div>
</div>
`;

exports[`only allows maxWrong incorrect guesses matches winning game: 2C 3C 4C > 5C 6C 1`] = `
<div>
  <div
    class="Snowman"
  >
    <img
      alt="0"
      src="0.png"
    />
    <p>
      Number wrong: 
      0
    </p>
    <p
      class="Snowman-word"
    >
      _
      _
      _
      _
      _
    </p>
    <p
      class=""
      id="letterButtonContainer"
    >
       
      <button
        value="a"
      >
        a
      </button>
      <button
        value="b"
      >
        b
      </button>
      <button
        value="c"
      >
        c
      </button>
      <button
        value="d"
      >
        d
      </button>
      <button
        value="e"
      >
        e
      </button>
      <button
        value="f"
      >
        f
      </button>
      <button
        value="g"
      >
        g
      </button>
      <button
        value="h"
      >
        h
      </button>
      <button
        value="i"
      >
        i
      </button>
      <button
        value="j"
      >
        j
      </button>
      <button
        value="k"
      >
        k
      </button>
      <button
        value="l"
      >
        l
      </button>
      <button
        value="m"
      >
        m
      </button>
      <button
        value="n"
      >
        n
      </button>
      <button
        value="o"
      >
        o
      </button>
      <button
        value="p"
      >
        p
      </button>
      <button
        value="q"
      >
        q
      </button>
      <button
        value="r"
      >
        r
      </button>
      <button
        value="s"
      >
        s
      </button>
      <button
        value="t"
      >
        t
      </button>
      <button
        value="u"
      >
        u
      </button>
      <button
        value="v"
      >
        v
      </button>
      <button
        value="w"
      >
        w
      </button>
      <button
        value="x"
      >
        x
      </button>
      <button
        value="y"
      >
        y
      </button>
      <button
        value="z"
      >
        z
      </button>
    </p>
  </div>
</div>
`;
